packages/app/README.md

```diff
@@ -1,51 +1,5 @@
 # @firebase/app
 
-This is the primary entrypoint to the Firebase JS SDK. **All apps using firebase
-will need this package**. Other `@firebase` packages are typically mounted on to
-the namespaces exposed by this package.
+This package coordinates the communication between the different Firebase components, as well as exposing the API surface to the user. 
```

Comment:
Just a small nit, but the tenses don't match here. It should be something like
> This package coordinates the communication between the different Firebase components, as 
> well as _exposes_ the API surface to the user.

I personally think it reads best as
> This package coordinates the communication between the different Firebase components and
> _exposes_ the API surface to the user.